README.md

```diff
@@ -53,13 +53,33 @@ http://localhost:8000/portal/service?id=95cf1a1b-c12e-4594-9d20-783988fe32b9
 
 Open a web browser and enter the URL. To log in, use one of the default users: `admin.user`, `normal.user` or `view.user`. The password is `test`.
 
-## Contributing
+## 3 Releases and Versioning
 
-Please read the [contribution guidelines](CONTRIBUTING.md) for this repository and keep our [code of conduct](CODE_OF_CONDUCT.md) in mind.
+Version numbers of _SoftiCAR Platform_ releases follow the [Semantic Versioning](https://semver.org/) principle.
 
-## Operations
+```
+     1.2.3
+    /  |  \
+major  |  patch
+     minor
+```
+
+1. If there was an **API break** since the previous release, the **major version** in incremented: `1.2.3 -> 2.0.0` -- API breaks include:
```

Comment:
```suggestion
1. If there was an **API break** since the previous release, the **major version** is incremented: `1.2.3 -> 2.0.0` -- API breaks include:
```